### PR TITLE
Mime config option support #1735 Fixed

### PR DESCRIPTION
**Pull request by comdiv**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 90c83c38717d74e2635d77587b4be3ff52cfc873
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b6f0a419f651e9352135048f6726979969684b5f
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: b824bc4025541b5f44a8fd1897bb48f8fcc6551c
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e62bff86dcdab2db8a57c69f530de00e5ff19928
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5d61436f0ee833dc366752cb7dd72c3911813ba5
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 24fc90eb396628b393fa60b2a279ddabb3e35596
  * Your commit message body contains a line that is longer than 80 characters

Guidelines are available at http://karma-runner.github.io/0.13/dev/git-commit-msg.html

---
This message was auto-generated by https://gitcop.com

**Comment by Dignifiedquire** [comment]
Thanks, will review tomorrow morning :)

**Comment by Dignifiedquire** [comment]
Please add some documentation for this as well.

File: test/unit/web-server.spec.js [hunk 1]
```diff
@@ -71,6 +73,10 @@ describe('web-server', () => {
       server = injector.invoke(m.createWebServer)
     })
 
+    it('should setup mime', () => {
```
Comment (Dignifiedquire) [inline test/unit/web-server.spec.js]: Would be good to have a test to ensure that the default mimes are still working

Comment (comdiv) [inline test/unit/web-server.spec.js]: OK, will add

**Comment by comdiv** [comment]
test added, config doc updated

**Comment by Dignifiedquire** [comment]
Thanks :octocat:

**Pull request merged**